main_windows.go

```diff
@@ -138,6 +166,10 @@ func runGraphicalUi() (err error) {
 
 		uiLog(address.String() + "> " + cmd)
 		sendRcon(cmd)
+		
+		// add to history
+		history = append(history, cmd)
+		historyIndex = len(history)
```

Comment:
It might be a good idea to limit the history array to a small number like 20, 50 or maybe 100 to avoid having it grow unnecessarily.